docs/content/deployment/guides/aws.mdx

```diff
@@ -78,6 +78,23 @@ def my_job():
 
 [Fargate tasks only support certain combinations of CPU and Memory.](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task-cpu-memory-error.html)
 
+### Customizing the Launched Run's Task
+
+By default, the <PyObject module="dagster_aws.ecs" object="EcsRunLauncher" /> uses the properties of the current ECS task to determine the configuration for each launched run's task. For example, the launched run will use the same ECS cluster, subnets, security groups, and launch type (e.g. Fargate or EC2) for each new task that it creates.
```

Comment:
Bullet 1 and bullet 3 are correct, bullet 2 is not. There's one ECS task per run, but those tasks don't launch other runs. The 'current or originating task' in this case is always a Dagster system task (usually the daemon)

Reply:
Hm - perhaps this will be fine for someone with more knowledge than myself, but my previous comment is how I interpreted this sentence:

> By default, the <PyObject module="dagster_aws.ecs" object="EcsRunLauncher" /> uses the properties of the current ECS task to determine the configuration for each launched run's task.

Reply:
New pass!

"The <PyObject module="dagster_aws.ecs" object="EcsRunLauncher" /> creates a new task for each run, using the current ECS task to determine network configuration. For example, the launched run will use the same ECS cluster, subnets, security groups, and launch type (e.g. Fargate or EC2)."

Reply:
I like that!